fault/magma_simulator_target.py

```diff
@@ -33,13 +47,17 @@ def run(self):
                     value = value.as_uint()
                 simulator.set_value(action.port, value)
             elif isinstance(action, actions.Print):
-                got = BitVector(simulator.get_value(action.port))
+                got = simulator.get_value(action.port)
+                if isinstance(action.port, (m.ArrayType, m.ArrayKind)) and \
```

Comment:
This PR only addresses arrays, which is blocking https://github.com/StanfordAHA/garnet/pull/105#issuecomment-422091599, can we save tuple support for separate PR?

Reply:
Not sure about Type and Kind, let me look into it, I think I saw other code that was like this so I was just duplicating the Type Kind check

Reply:
Actually, it's not just `isinstance(action.port, (m.BitsType, m.BitsKind))`, because 
```
>>> a = m.Array(2, m.Bit)
>>> isinstance(a, m.BitsKind)
False
```
This allows you to print a simple array of bits (that wasn't explicitly a bits type)